tests/python/topi/python/test_topi_lrn.py

```diff
@@ -66,6 +65,7 @@ def test_lrn():
     verify_lrn((1, 3, 5, 5), 3, 1, 1.0, 1.0, 0.5)
     verify_lrn((1, 3, 5, 5), 3, 3, 1.0, 1.0, 0.5)
     verify_lrn((1, 3, 20, 20), 3, 1, 2.0, 1.0, 0.75)
+    verify_lrn((1, 3, 5, 5), 3, 3, 1.0, 1.0, 0.5, dtype="float16", rtol=1e-3, atol=1e-3)
```

Comment:
Do you mean the 1e-3 tolerance doesn’t work for this set of parameters?

Reply:
Yes, It seems that original rtol 1e5 not work with f16. It failed on my machine like
```
Mismatched elements: 4 / 75 (5.33%)
Max absolute difference: 0.000977
Max relative difference: 0.001402
 x: array([[[[2.761e-01, 3.909e-01, 2.979e-01, 1.460e-01, 1.453e-01],
         [7.842e-01, 7.104e-01, 3.000e-01, 7.490e-01, 5.640e-01],
         [1.940e-01, 1.163e-01, 6.709e-01, 1.050e-01, 5.229e-01],...
 y: array([[[[2.761e-01, 3.909e-01, 2.979e-01, 1.460e-01, 1.453e-01],
         [7.842e-01, 7.100e-01, 3.000e-01, 7.490e-01, 5.640e-01],
         [1.940e-01, 1.163e-01, 6.709e-01, 1.050e-01, 5.229e-01],...
```

Reply:
Got it. Thanks for the explanation!